tests/fireflyClient.test.ts

```diff
@@ -63,20 +65,24 @@ describe("FireflyClient", () => {
 
     // market data
     const marketData = await client.getMarketData(symbol);
+    console.log(marketData.data)
     if (marketData.data && bnStrToBaseNumber(marketData.data.marketPrice) > 0) {
-      const marketPrice = bnStrToBaseNumber(marketData.data.marketPrice);
+      marketPrice = bnStrToBaseNumber(marketData.data.marketPrice);
+      indexPrice = bnStrToBaseNumber(marketData.data.indexPrice || "0")
       const percentChange = 3 / 100; // 3%
       buyPrice = Number((marketPrice - marketPrice * percentChange).toFixed(0));
       sellPrice = Number(
         (marketPrice + marketPrice * percentChange).toFixed(0)
       );
       console.log(`- market price: ${marketPrice}`);
```

Comment:
remove logs

Reply:
These logs are added so that user knows what was buy/sell order price as the test uses these prices to place an order